Source/UI/Controllers/LimeAuthUIBaseViewController.swift

```diff
@@ -90,4 +90,13 @@ open class LimeAuthUIBaseViewController: UIViewController {
     
     /// If controller's background was configured with image, then this property contains a view with that image.
     public private(set) weak var backgroundImageView: UIImageView?
+    
+    // MARK: - Others
+    
+    /// Sets if "swipe down to dismiss" gesture should be enable when presented modally on iOS 13 and later
+    public func setGestureDismissEnabled(to enabled: Bool) {
```

Comment:
Just a nitpick. I would rename it to something like `setSwipeToDismissGestureEnabled(to enabled: Bool)`

Reply:
Just one note. Perhaps we should keep the original value of `(navigationController ?? self).isModalInPresentation` and restore it in case of "enable". Something like this:
```swift
private var isModalInPresentationOldValue: Bool?

public func disableSwipeToDismissGesture() {
    if #available(iOS 13.0, *) {
        guard isModalInPresentationOldValue == nil else {
            return
        }
        isModalInPresentationOldValue = (navigationController ?? self).isModalInPresentation
        (navigationController ?? self).isModalInPresentation = true
    }
}

public func restoreSwipeToDismissGesture() {
    if #available(iOS 13.0, *) {
        guard let odlValue = isModalInPresentationOldValue else {
            return
        }
        (navigationController ?? self).isModalInPresentation = isModalInPresentationOldValue
        isModalInPresentationOldValue = nil
    }
}
```

Reply:
Good idea.

Reply:
I improved the API so it's easier to use without any additional variable